db/schema.rb

```diff
@@ -225,7 +247,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "archived_on"
-    t.bigint "tenant_id"
+    t.bigint "tenant_id", null: false
```

Comment:
hm, I didn't push these in my previous migration (we are missing the check)

Reply:
Yeah we need to re-run db:migrate to write out the db/schema.rb file after changing a migration